build/tasks/task-browser-frontend.js

```diff
@@ -60,6 +65,7 @@ gulp.task('browser-frontend:webpack', () => new Promise((resolve, reject) => {
 gulp.task('browser-frontend:build', gulp.series(
   'browser-frontend:copy-html',
   'browser-frontend:webpack',
+  'browser-frontend:copy-qbrt-shell',
```

Comment:
Nit: So I guess we're doing this even for electron backends, which is probably fine. I'd have the same remark to make for packaged builds, where we bundle the dummy and qbrt runners as well, even though we don't have to. Not important right now.